webapp/src/utils.ts

```diff
@@ -112,8 +112,14 @@ class Utils {
     static htmlFromMarkdown(text: string): string {
         // HACKHACK: Somehow, marked doesn't encode angle brackets
         const renderer = new marked.Renderer()
-        if ((window as any).openInNewBrowser) {
-            renderer.link = (href, title, contents) => `<a target="_blank" rel="noreferrer" href="${encodeURI(href || '')}" title="${title ? encodeURI(title) : ''}" onclick="event.stopPropagation(); openInNewBrowser && openInNewBrowser(event.target.href);">${contents}</a>`
+        renderer.link = (href, title, contents) => {
+            return '<a ' +
+                'target="_blank" ' +
+                'rel="noreferrer" ' +
+                `href="${encodeURI(href || '')}" ` +
+                `title="${title ? encodeURI(title) : ''}" ` +
+                'onclick="event.stopPropagation(); openInNewBrowser && openInNewBrowser(event.target.href);"' +
```

Comment:
Nice change :)

Reply:
this `openInNewBrowser` was something for the desktop app, and allowing this everywhere is not needed, and a bit less robust that adding just the `target='_blank'`.

Reply:
@jespino I've updated this to use `openInNewBrowser` only when available. Should be good now.